File: public/phase-2-20160620022203/C_SHARP/two-piles-of-balls.cs
// Name : Two Piles of Balls
using System;
namespace Application {
	class two_pilesof_balls {
		public static void Main(string[] args) {
		// input:
		// n : number of balls in a pile
		// a : 
		int index;
		int n = Convert.ToInt32(Console.ReadLine());
		int[] a = new int[2n];
		string[] a_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<2n;index++)
			a[index] = Convert.ToInt32(a_elements[index]);


		// write your code here
		// store your results in `result`,`b`

		// output
		// Dummy Data
		int result=1;
		int[] b = new int[] {2,1};

		Console.WriteLine(result);
		Console.WriteLine(string.Join(" ", b));
		}	
	}
}

File: public/phase-2-20160620022203/C/two-piles-of-balls.c
// Name : Two Piles of Balls
#include <stdio.h>
int main() {
	// input:
	// n : number of balls in a pile
	// a : 
	int n, a[100], result=1, b[100]={2,1}, index;
	scanf("%d", &n);
	for(index = 0; index< 2n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `result`,`b`

	// output
	printf("%d", result);
	for(index = 0; index< (2*n); index++) {
		printf("%d ",b[index]);
	}
	return 0;
}